coordinator/src/services/translation/request/task.ts

```diff
@@ -2,17 +2,20 @@ import {fhir, hl7V3} from "@models"
 import {getIdentifierValueForSystem} from "../common"
 import * as pino from "pino"
 import {createAuthorForUnattendedAccess} from "./agent-unattended"
+import Hapi from "@hapi/hapi"
 
 export async function createAuthorFromTaskOwnerIdentifier(
-  identifier: fhir.Identifier,
+  task: fhir.Task,
```

Comment:
Is there a reason we're passing in the whole Task now? Not hugely bothered but the intention was to pass in as little as possible to make testing easier